recipe/meta.yaml

```diff
@@ -8,45 +8,37 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.bz2
   url: https://sourceforge.net/projects/qwt/files/qwt/{{ version }}/{{ name }}-{{ version }}.tar.bz2
-  sha256: 1529215329e51fc562e0009505a838f427919a18b362afff441f035b2d9b5bd9
+  sha256: 2b431a6e04cb939f71cdc03a343975162bd6f94bfc11b85227bdb44e9576f5ba
   patches:
     # Can't seem to avoid /usr/local/ install without patch
     - qwtconfig.pri.patch
-    # build in release mode
-    - windows.patch
 
 build:
-  number: 1001
+  number: 0
 
 requirements:
   build:
-    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ cdt('mesa-libgl-devel') }}  # [linux]
-    - {{ cdt('mesa-dri-drivers') }}  # [linux]
-    - {{ cdt('libselinux') }}  # [linux]
-    - {{ cdt('libxdamage') }}  # [linux]
-    - {{ cdt('libxfixes') }}  # [linux]
-    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ compiler('c') }}
```

Comment:
Please, let the recipe equal to the master, just change the version which you want

Reply:
Ops! Will do